03_Modules/Configuration/src/module/BootNotificationService.ts

```diff
@@ -66,7 +66,7 @@ export class BootNotificationService {
       }
     }
 
-    return bootStatus;
+    return OCPP2_0_1_Mapper.BootMapper.toRegistrationStatusEnumType(bootStatus);
```

Comment:
Very nitpick, and me just thinking out loud. Would making sure on line 39 that the bootStatus is a `RegistrationStatusEnumType` and from there on out operate on that assumption. Then we have a nicer return type. This just came about because I was looking at `bootStatus = OCPP2_0_1.RegistrationStatusEnumType.Accepted;` why are we putting it through the mapper again...